main.cpp

```diff
@@ -117,6 +120,91 @@ Thing 10) Audio Mixer
     3) monitor input through headphones 
  */
 
+/*
+Thing 5) Equalizer
+5 properties:
+    1) high pass switch 
+    2) low frequency controls
+    3) low-mid frequency controls
+    4) mid frequency controls
+    5) high frequency controls
```

Comment:
these 'controls' normally consist of several smaller sub-controls. i.e. low frequency controls would include:
- slope
- gain 
- frequency
- peak or shelf
Things 5-9 are not supposed to have sub-objects as their properties.  
Please revise your 5-9 so that all of the properties are easily convertible to primitives
I.e. `lowFrequencyHz` or `low mid gain level`
 - [ ] revise